uint8 left
uint8 middle
uint8 right
uint8 rearLeft
uint8 rearRight

